python/lsst/cp/pipe/ptc.py

```diff
@@ -437,28 +336,30 @@ def runDataRef(self, dataRefList):
             # Fill up PhotonTransferCurveDataset object.
             datasetPtc = self.fitPtc(datasetPtc, self.config.ptcFitType)
 
+        detName = detector.getName()
+        now = datetime.datetime.utcnow()
+        calibDate = now.strftime("%Y-%m-%d")
+        butler = dataRef.getButler()
+        datasetPtc.updateMetadata(setDate=True, detectorName=detName, detectorId=detector.getId(),
```

Comment:
`updateMetadata` now supports passing the camera and detector directly, which simplifies this a bit.